quickstarts/host-inventory-quickstart.yaml

```diff
@@ -0,0 +1,70 @@
+  introduction: |-
+    ### This quick start shows you how to get started with your on-premise host inventory.
+    Hosts are discovered and stored in infrastructure environments. You can use the console to create an infrastructure environment to manage your hosts and create clusters on those hosts. You can manage your host inventory with infrastructure environments, which allow you to set the same configurations for all hosts that it contains.
+
+
+  tasks:
+    - description: >-
+
+        1.  Configure your host inventory settings. View [documentation](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.5/html-single/clusters/index#enable-cim) to follow the steps to configure your host inventory settings.
+
+        1. Create an infrastructure environment for your host inventory.
+
+          a. Go to **Host inventory**
+
+          b. Select **Create infrastructure environment**
+
+        1. Add hosts.
+
+          a. In the infrastructure that you created, choose one of the **Add hosts** options
+
+          b. After you successfully added your hosts, ensure the status of each host is available for cluster creation.
+
+      review:
+        failedTaskHelp: This task isn’t verified yet. Try the task again.
+        instructions: >-
+          Verify that the on-premise host inventory was successfully prepared:
+
+          1. From the navigation menu, click **Infrastructure**, then select **Host Inventory**.
+
+            a. Does the table list your created infrastructure environment?
+
+          1. From the table, select your created infrastructure environment, then select the **Hosts** tab.
+          
+            a. From the **Status** menu, are your hosts listed as "available"?
+          
+          
+      summary:
+        failed: Try the steps again.
+        success: >-
+          Charts from the **myChartRepo** repository are now available.
+      title: Get started with creating an on-premise host inventory
+  conclusion: >-
+    You configured your host inventory and can now create a cluster.
+
+    1. Click **Create cluster**
+
+    2. Select **Host Inventory**
+
+    3. Choose either **Hosted control plane** or **Standalone control plane**
+
+    4. Follow the remaining steps of the cluster creation process
```

Comment:
Here we are using punctuation at times, and then not at times. All complete sentences should have a period at the end, like the questions and other statements do. (Example, like 68)

Reply:
the only other thing I will mention is we don't use quotation marks in doc for accessibility reasons, I am not sure how the UI highlighting works, though.
[10:45](https://coreos.slack.com/archives/D013DPB9RTK/p1669736755197489)
and if the OCP team is using it, then I say go forth but we may want to bring that up.

Reply:
https://www.patternfly.org/v4/ux-writing/punctuation#quotation-marks